gunicorn/arbiter.py

```diff
@@ -494,6 +496,11 @@ def manage_workers(self):
 
     def spawn_worker(self):
         self.worker_age += 1
+        worker_id = self.num_workers
+        for i in range(self.num_workers):
```

Comment:
I was thinking that initializing worker_id to self.num_workers could handle some weird edge cases where a worker is started but there isn't an unclaimed id in range(0, self.num_workers). Not sure if such a case would be possible though, I haven't been able to make that happen in any of my testing.